docker-compose.yml

```diff
@@ -10,31 +11,40 @@ services:
       timeout: 3s
       retries: 5
     env_file:
-      - ./server/.env
+      - path: ./server.env
+        required: false
+      - ./server/.env.example
```

Comment:
Isn't this in order? Top to bottom?

So the example will always overwrite the previous...

This is a bit wrong 😅

Reply:
> I would kinda prefer that you copy the .Env file like I did in my pr

agreed